### DISCUSSION TITLE
Meeting Minutes 04-05-2023

### DISCUSSION THREAD

**Discussion by tmorshed** [NeuroTechX/EEG-ExPy#231] [Developer's Meetings]
**Attendees:** @tmorshed @oreHGA @pellet @retiutut 

## Updates:
**Ben:**
- Great progress, able to review for quite a bit now.
- Sent it to JohnGrif.
  - He had some problems with the VR headset but most things worked
  - **Suggestion from Ore:** Get a PR going
 
**Ore:**
- Cleaned out the backlog of issues
- The build pipelines work
  - Type checks not working for some of the things
- Let's close the rest of stale issues - and fix things as we go.
  - Let's get to 0 issues by the end of summer. Currently have 30. Some need action, some are stale.
- Versioning and pushing - finish the `dev` > `master` merge.
- Suggestion: analytics, to see how people use `eeg-notebooks`
   - With privacy in mind. To guide further development

**Taha:** 
- Will be working on two issues:
  - `np.binomial` issue for oddball tasks
  - Event order through experimentation

**Richard:**
- Let's get the VR work to a good PR
- In the next steps, let's integrate that with Galea
- Also a maintainer at `brainflow` now. Can work on bridging gaps in libraries.
- Interested in the brainflow-related issues. (@oreHGA will send it ffr)

**Comment by pellet** [comment]
John had a lot of problems with getting the quest 2 to link to his PC, so not sure if he's been able to test the N170 experiment yet but it looks mostly good from the data I had collected I think?  The stimulus is a little shaky, so I may need to smooth out the movement of the stimulus shown in front of the face.
